pkg/testcase/cluster.go

```diff
@@ -114,3 +114,30 @@ func checkAndPrintAgentNodeIPs(agentNum int, agentIPs []string, isWindows bool)
 		Expect(agentIPs).Should(BeEmpty())
 	}
 }
+
+func ClusterReset() {
```

Comment:
if this is supposed to be a test case should follow convention 
Test<nameofYourTest>